### PR TITLE
Rework tree tab a little

### PR DESCRIPTION
**Pull request by McSinyx**
First stab at GH-77.  I'm still battling with nested fragment.

### CODE REVIEW

File: app/src/main/java/vn/edu/usth/minigh/fragments/TreeFragment.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import vn.edu.usth.minigh.R
+import vn.edu.usth.minigh.api.github
+
+class TreeNodeFragment : Fragment(R.layout.fragment_tree_node) {
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        val args = requireArguments()
+        // TODO: handle symlinks and git submodules
+        val type = if (args.getString("type") == "dir") {
+            R.string.fa_folder
```
Comment (Huy-Ngo) [inline app/src/main/java/vn/edu/usth/minigh/fragments/TreeFragment.kt:27]: Please use [solid folder](https://fontawesome.com/icons/folder?style=solid) for more visual distinction.

**Review by Huy-Ngo — APPROVED** [review]
K, it looks very nice now.

File: app/src/main/java/vn/edu/usth/minigh/fragments/TreeFragment.kt (hunk 1 above)
Comment (McSinyx) [reply app/src/main/java/vn/edu/usth/minigh/fragments/TreeFragment.kt]: Addressed in e1e476a.

**Comment by McSinyx** [comment]
Thanks!

**Pull request merged**